packages/rest/README.md

```diff
@@ -12,18 +12,17 @@ with:
 - new custom routing engine (special thanks to @bajtos)!
 - tools for defining your application routes
 - OpenAPI 2.0 spec (`swagger.json`/`swagger.yaml`) generation
-- a default sequence implementation to manage the request and response lifecycle
+- a default sequence implementation to manage the request and response life cycle
```

Comment:
what's wrong with "lifecycle"? :)

Reply:
Apparently `life cycle` is the more correct usage / grammar as per online dictionaries and a VSCode extension I installed. That said, I can revert this change (I prefer `lifecycle` as well).

Reply:
Go with whatever you think that's correct. 
i originally thought (coming from my son's learning material) `life cycle` is more like some biological cycle that living things go through.  But i'm no english expert. :)